sandbox/registration-processor-mz.properties

```diff
@@ -677,6 +702,16 @@ mosip.regproc.printing.server.port=8099
 mosip.regproc.printing.server.servlet.path=/registrationprocessor/v1/print-stage
 mosip.regproc.printing.eventbus.port=5722
 
+#workflow-manager-service
+mosip.regproc.workflow.action.job.eventbus.kafka.commit.type=single
```

Comment:
Keep all the workflow-manager properties together

Reply:
fixed